.jenkins/caffe2/build.sh

```diff
@@ -4,12 +4,14 @@ set -ex
 
 source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
 
-# TODO: Migrate all centos jobs to use proper devtoolset
-if [[ "$BUILD_ENVIRONMENT" == *py2-cuda9.0-cudnn7-centos7* ]]; then
-  # There is a bug in pango packge on Centos7 that causes undefined
-  # symbols, upgrading glib2 to >=2.56.1 solves the issue. See
-  # https://bugs.centos.org/view.php?id=15495
-  sudo yum install -y -q glib2-2.56.1
+if [[ "$BUILD_ENVIRONMENT" == *centos7* ]]; then
```

Comment:
This is not the right way to do it. We have a way to specify devtoolset in CentOS environments https://github.com/pytorch/pytorch/blob/master/docker/caffe2/jenkins/common/install_devtoolset.sh.

Reply:
@bddppq should we revert this?